MFAssignR/tests/testthat/test-MFAssign.R

```diff
@@ -0,0 +1,41 @@
+expected_paths <- function(mode) {
+  return(list(
+    unambig  = file.path("test-data", paste0(mode, "_unambig.rds")),
+    ambig  = file.path("test-data", paste0(mode, "_ambig.rds")),
+    none  = file.path("test-data", paste0(mode, "_none.rds"))
+  ))
+}
+
+load_expected <- function(mode) {
+  paths <- expected_paths(mode)
+  return(list(
+    Unambig = readRDS(paths$unambig),
+    Ambig = readRDS(paths$ambig),
+    None = readRDS(paths$none)
+  ))
+}
+
+save_expected <- function(actual, mode) {
+  paths <- expected_paths(mode)
+  saveRDS(actual$Unambig, paths$unambig)
+  saveRDS(actual$Ambig, paths$ambig)
+  saveRDS(actual$None, paths$none)
+}
+
+patrick::with_parameters_test_that("MFAssign works",
+  {
+    peaks <- readRDS(file.path("test-data", paste0(mode, "_iso.rds")))
+    actual <- MFAssign(
+      peaks = peaks$Mono,
+      isopeaks = peaks$Iso,
+      ionMode = mode
+    )
+    save_expected(actual, mode)
```

Comment:
Are we saving the expected file on each run?

Reply:
Good catch!